integration-tests/tests/run-workflow.js

```diff
@@ -58,22 +59,27 @@ const launchWorkflowAndWaitForSuccess = async page => {
   ])
   // stopLoggingPageAjaxResponses()
 
-  const disablePageLogging = logPageConsoleMessages(page)
```

Comment:
I misunderstood the point of this method… it is not for logging related to tests, but console logs from Terra UI itself. So removing this usage and switching to `rawConsole` to lessen noise in the tests.